imap_processing/ena_maps/ena_maps.py

```diff
@@ -412,10 +430,40 @@ def __init__(
                 self.sky_grid.el_grid.ravel(),
             )
         )
-        self.num_points = self.az_el_points.shape[0]
 
 
-class UltraPointingSet(PointingSet):
+class HealPixPointingSet(PointingSet, ABC):
+    """
+    Abstract base class for HealPix pointing sets.
```

Comment:
Nitpick: You usually use HEALPix, rather than HealPix here. I have a hard time with mixed case, so I think it's arguably better to use Healpix, with just the H capitalized, in documentation/longer comments. I think lowercase healpix is also fine for short comments. 

Note: I'm sure I've strayed from this rule myself! It's just easier to see in unfamiliar comments.